src/Microsoft.Identity.Web.Diagnostics/IdHelper.cs

```diff
@@ -6,10 +6,12 @@
 using System.Reflection;
 using System.Text.RegularExpressions;
 
-namespace Microsoft.Identity.Web
+namespace Microsoft.Identity.Web.Diagnostics
```

Comment:
why?

Reply:
I'll revert it, but some files are in the diagnostics namespace and some are in the "id.web" namepace.